core/src/banking_stage.rs

```diff
@@ -2054,16 +2058,16 @@ impl BankingStage {
             .fetch_add(newly_buffered_packets_count, Ordering::Relaxed);
         banking_stage_stats
             .current_buffered_packet_batches_count
-            .swap(buffered_packet_batches.len(), Ordering::Relaxed);
+            .swap(unprocessed_transaction_storage.len(), Ordering::Relaxed);
```

Comment:
I agree, probably would want to divide by `batch_size` here, not sure what the original intent of this metric was.

Reply:
seems at some point we had 
```
banking_stage_stats.current_buffered_packets_count.swap(
           buffered_packets.iter().map(|packets| packets.1.len()).sum(),
           Ordering::Relaxed,
);
```
because back then we stored the packets as batches, however now we use individual packets in the minmaxheap.

I think that means we can deprecate one of these metric points.

Reply:
agree too, time to deprecate some metrics, can only back-fitting them for so long :)

Reply:
yeah agreed, this can be deleted, as well as the corresponding field in the datapoint. The `current_buffered_packets_count` below this covers the number of packets in the buffer already